src/development/going-live.md

```diff
@@ -1,103 +1,89 @@
 
 > **note**
 > If you are on a Development plan, you cannot add a domain. You would need to upgrade your subscription to a production plan.
 
-## 2 - Routes
 
-You can configure the routes of your project directly within the
-Web Interface or within your `.platform/routes.yaml` file.
+## Configure your DNS provider
 
-### Single hostname
+Configure your DNS provider to point your domain to your Platform.sh Master environment.
 
-If you want your site to only be available at `http://mydomain.com` and
-have `http://www.mydomain.com` redirect to `http://mydomain.com`, you
-need define your `routes.yaml` as follow:
+The way to do so will vary somewhat depending on your registrar, but nearly all registrars should allow you to set a CNAME.  Some will call it an Alias or similar alternate name, but either way the intent is to say "this domain should always resolve to... this other domain".  Add a CNAME record from your desired domain (`example.com`) to the master environment hostname you wrote down earlier.
 
-```yaml
-"http://{default}/":
-    type: upstream
-    upstream: "app:http"
+If you use multiple domain names for your site, such as `www.example.com` and `example.com`, you'll need to add a CNAME for each of them.
 
-"http://www.{default}/":
-    type: redirect
-    to: "http://{default}/"
-```
+Note that depending on your registrar and the TTL you set, it could take anywhere from 15 minutes to 12 hours for the DNS change to fully propagate across the Internet.
 
-### Multiple hostnames
+## Example setup
 
-If you want your site to be available at both `http://mydomain.com` and
-`http://www.mydomain.com`, you need to define one upstream for each
-hostname.
+For example, suppose your project ID is `abc123` in the US region, and you've registered `mysite.com`.  You want `www.mysite.com` to be the "real" site and `mysite.com` to redirect to it.
 
-Here would be an example of your `routes.yaml` for the
-`http://mydomain.com` URL:
+### Configure `routes.yaml`
 
-```yaml
-"http://{default}/":
-    type: upstream
-    upstream: "app:http"
+First, configure your `routes.yaml` file like so:
 
+```yaml
 "http://www.{default}/":
-    type: upstream
-    upstream: "app:http"
+  type: upstream
+  upstream: "app:http"
+"http://{default}/":
+  type: redirect
+  to: "http://www.{default}/"
 ```
 
-> **note**
-> You can test those routes on your development environments with:
-> * `http://[branch]-[project-id].[region].platform.sh`
-> * `http://www-[branch]-[project-id].[region].platform.sh`
+That will result in two domains being created on Platform.sh: `master-def456-abc123.us.platform.sh` and `www---master-def456-abc123.us.platform.sh`.  The former will automatically redirect to the latter.  In the `routes.yaml` file, `{default}` will automatically be replaced with `master-def456-abc123.us.platform.sh`.  In domain prefixes (like `www`), the `.` will be replaced with `---`.
 
-### HTTPS
+### Set your domain
```

Comment:
This is a 3-rd level section; the previous is 2nd level.  The intent is to mirror the section above with a concrete, non-abstract example.